pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -860,6 +862,55 @@ func getSRIOVPCIAddresses(ifaces []v1.Interface) map[string][]string {
 	return networkToAddressesMap
 }
 
+// This function parses all environment variables with prefix GPU_PASSTHROUGH_DEVICES that are set by
+// device plugins. It lists PCI IDs for devices allocated to the pod. The format
+// is as follows:
+// "": for no allocated devices
+// "0000:81:11.1,": for a single device
+// "0000:81:11.1,0000:81:11.2[,...]": for multiple devices
+func getGpuAddresses() []string {
```

Comment:
as I see `getGpuAddresses` and `getVgpuAddresses` are essentially identical with a difference of the `(v)gpuEnvPrefix` prefix. Perhaps these two can be merged into one function? Maybe something like:
`func getEnvAddressListByPrefix(evnPrefix string) []string {`

Reply:
Updated